ironmon_tracker/data/MoveData.lua

```diff
@@ -135,6 +135,93 @@ MoveData.TypeToEffectiveness = {
 	steel = { fire = 0.5, water = 0.5, ice = 2, rock = 2, steel = 0.5, electric = 0.5 },
 }
 
+-- Individual formulas for calculating any changes to a move based on contextual information, such as being in battle
+MoveData.VariablePowerAdjustmentFuncs = {
+	[MoveData.Values.WeatherBallId] = function(move, sourcePokemon, targetPokemon)
+		if not Battle.inActiveBattle() then return end
```

Comment:
this return is redundant as there's already an early return in `Utils.calculateWeatherBall`, though arguably the latter one in the Utils func is made redundant by this one and could be removed

No harm keeping both of course, just mentioning as a small bit of clean-up that could be done